frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_node_widget.dart

```diff
@@ -333,49 +360,40 @@ class _SmartEditInputState extends State<_SmartEditInput> {
   }
 
   Future<void> _requestCompletions() async {
-    final result = await UserBackendService.getCurrentUserProfile();
-    return result.fold((l) async {
-      final openAIRepository = HttpOpenAIRepository(
-        client: client,
-        apiKey: l.openaiKey,
-      );
+    final openAIRepository = await getIt.getAsync<OpenAIRepository>() as HttpOpenAIRepository;
```

Comment:
I worry about this cast here. Is it not possible to `getStreamedCompletions` on `OpenAIRepsoitory`?

Reply:
`getStreamdCompletions` just needs to be updated inside the abstract class `OpenAIRepository` to include the named parameter `bool useAction = false,` from what I can tell.